auth-api/src/main/java/io/stargate/auth/api/impl/AuthApiServerFactory.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.auth.api.impl;
+
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import io.dropwizard.jetty.ContextRoutingHandler;
+import io.dropwizard.server.SimpleServerFactory;
+import io.dropwizard.setup.Environment;
+import java.util.Collections;
+import java.util.Map;
+import org.eclipse.jetty.server.Connector;
+import org.eclipse.jetty.server.Handler;
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.util.thread.ThreadPool;
+
+/**
+ * @see "META-INF/services/io.dropwizard.server.ServerFactory"
+ * @see "config.yaml"
+ */
+@JsonTypeName("auth-api")
+public class AuthApiServerFactory extends SimpleServerFactory {
```

Comment:
The admin resources were still exposed under `/admin` (default path), this completely removes them.

It's the exact same class for all 3 modules. I would have liked to factor it in `core`, but this introduces more issues with OSGi 😞

Reply:
What kind of issues? :)

We could create a new module (e.g. `web-common`) and produce a _plain_ jar from it that would then be _embedded_ into the various OSGi bundles (unlike `core`, which is a bundle in itself). I believe this way would give us ability to reuse code without introducing dependency problems.

Reply:
I've created a follow-up ticket: #439.